aws/data_source_aws_instance.go

```diff
@@ -93,6 +93,11 @@ func dataSourceAwsInstance() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"ipv6_addresses": {
+				Type:     schema.TypeList,
```

Comment:
Unless the order of the returned IPv6 addresses is significant, we prefer `schema.TypeSet` for collections.

Reply:
changed to TypeList

Reply:
whoops, i meant change to TypeSet as requested